src/screens/ProfileScreen/index.js

```diff
@@ -640,7 +641,6 @@ const ProfileScreen = ({route}) => {
   };
 
   const onDeletePost = async () => {
-    setIsOptionModalOpen(false);
     removePostByIdFromContext();
 
     let response;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `onDeletePost` has been modified to remove the call to `setIsOptionModalOpen(false)`. If this state update was responsible for closing a modal after deleting a post, its removal might cause the modal to stay open even after the post is deleted.
```diff
- setIsOptionModalOpen(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->